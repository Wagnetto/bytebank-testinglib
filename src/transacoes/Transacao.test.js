import { render } from "@testing-library/react";
import React from "react";
import Transacao from "./Transacao";

describe("Transaction Component", () => {
  it("transaction napshot should keep the same", async () => {
    const { container } = render(
      <Transacao data="08/09/2020" tipo="saque" valor="20.00" />
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
