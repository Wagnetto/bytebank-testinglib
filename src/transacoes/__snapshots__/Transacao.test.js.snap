// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Transaction Component transaction napshot should keep the same 1`] = `
<div
  class="transacao-container"
>
  <p>
    08/09/2020
  </p>
  <p>
    saque
  </p>
  <p>
    R$ 
    20.00
  </p>
</div>
`;
